artemis/src/main/java/tech/pegasys/artemis/BeaconNode.java

```diff
@@ -88,18 +88,8 @@
   public void start() {
 
     try {
-      this.serviceConfig.getConfig().validateConfig();
       metricsEndpoint.start();
-      // Initialize services
-      serviceController.initAll(
-          serviceConfig,
-          BeaconChainService.class,
-          PowchainService.class,
-          ChainStorageService.class);
-
-      // Start services
-      serviceController.startAll();
-
+      serviceController.start().reportExceptions();
```

Comment:
Should we be doing a `join` here? Then if any component fails to start we throw an exception back up to BeaconNodeCommand (we should remove the try/catch in this class).  That in turn lets PicoCLI be notified of the failure properly.

Reply:
Good call 👍